.github/workflows/continuous-benchmarking-runtimes.yml

```diff
@@ -3,7 +3,7 @@ name: Continuous benchmarking - Runtime experiments
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 0 * * *"
+    - cron: "0 0 * * 2"
```

Comment:
Is the reason for staggering experiments to prevent timeouts? (Assuming we are only going to setup/teardown 1 VM on-demand for providers moving forward) This might be important information that needs to be put on the dashboard since the experiment times are now different.

Reply:
Yup, if there is only one runner for each provider, I was concerned that the jobs might timeout if they are queued for too long, will confirm the schedule with Dmitrii

I guess the other option is to provision multiple VMs to run all of them simultaneously